docs/plugin_developers_guide/02_writing_plugins.rst

```diff
@@ -57,14 +57,15 @@ A typical Python implementation of this would simply return a fixed dictionary o
 
 These are the properties returned by the JSON document:
 
-- **Name** - A textual name that will be used for reporting purposes for this plugin.
-- **Version** - This property allows the version of the plugin to be communicated to the plugin loader. This is used for reporting purposes only and has no effect on the way Fledge interacts with the plugin.
-- **Type** - The type of the plugin, used by the plugin loader to determine if the plugin is being used correctly. The type is a simple string and may be South, North, Storage, Filter, Rule or Delivery.
+- **name** - A textual name that will be used for reporting purposes for this plugin.
+- **version** - This property allows the version of the plugin to be communicated to the plugin loader. This is used for reporting purposes only and has no effect on the way Fledge interacts with the plugin.
+- **mode** - A set of options the define how the plugin operations, these are separated from each other using the | symbol.
+- **type** - The type of the plugin, used by the plugin loader to determine if the plugin is being used correctly. The type is a simple string and may be South, North, Storage, Filter, Rule or Delivery.
```

Comment:
lower case? (south, north, filter). Not sure about storage, rule, delivery types, Do we have support of these?

Reply:
We don't support storage, otherwise yes we do for the others.